emission/integrationTests/nominatimTests/nominatim-docker-test.yml

```diff
@@ -0,0 +1,34 @@
+name: nominatim-docker-test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches:
+        - master
+        - nominatim
+  pull_request:
+    branches:
+        - master
+
+  schedule:
+    # Run every Sunday at 4:05 am
+    - cron:  '5 4 * * 0'
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout
+    - uses: actions/checkout@v2
+
+    # Runs a single command using the runners shell
+    - name: Workflow test
+      run: echo Smoke test
+
+    - name: Initialize nominatim 
+      run: docker-compose -f docker-compose.yml up --exit-code-from web-server
```

Comment:
this is wrong docker-compose file you need to integrationTests one

Reply:
to basic test the GitHub action, you should run the steps after the checkout job in the directory where you have checked out your code.